lld/ELF/ICF.cpp

```diff
@@ -324,6 +324,8 @@ bool ICF<ELFT>::equalsConstant(const InputSection *a, const InputSection *b) {
 
   const RelsOrRelas<ELFT> ra = a->template relsOrRelas<ELFT>();
   const RelsOrRelas<ELFT> rb = b->template relsOrRelas<ELFT>();
+  if (ra.areRelocsCrel())
```

Comment:
Should this also have `|| rb.areRelocsCrel()` like the rel case below?

Reply:
Not needed. The double rel check is for 686cff17cc310884e48ae963bf7507f96950cc90 .

An early `areRelocsCrel()` issue would not run into that issue.